elasticsearch5/src/main/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndex.java

```diff
@@ -78,6 +78,7 @@
     public static final String ELEMENT_TYPE_FIELD_NAME = "__elementType";
     public static final String VISIBILITY_FIELD_NAME = "__visibility";
     public static final String HIDDEN_VERTEX_FIELD_NAME = "__hidden";
+    public static final String HIDDEN_PROPERTY_FIELD_NAME = "__hidden_property";
```

Comment:
I'm guessing we don't need to add it here because it will always have visibility correct? https://github.com/visallo/vertexium/blob/a384cebded60878f47d3e00084473e7d255616e1/elasticsearch5/src/main/java/org/vertexium/elasticsearch5/Elasticsearch5SearchIndex.java#L1442